pom.xml

```diff
@@ -61,22 +61,22 @@
 
     <properties>
         <assertj-core.version>3.20.2</assertj-core.version>
-        <jackson.version>2.12.5</jackson.version>
+        <jackson.version>2.12.6</jackson.version>
         <jersey.version>2.35</jersey.version>
-        <jetty.version>9.4.43.v20210629</jetty.version>
-        <junit.version>4.13.1</junit.version>
+        <jetty.version>9.4.44.v20210927</jetty.version>
+        <junit.version>4.13.2</junit.version>
         <junit-jupiter.version>5.7.2</junit-jupiter.version>
         <mockito-junit-jupiter.version>3.12.4</mockito-junit-jupiter.version>
-        <logback.version>1.2.5</logback.version>
+        <logback.version>1.2.11</logback.version>
         <logback-json.version>0.1.5</logback-json.version>
         <mockito.version>3.11.2</mockito.version>
-        <netty.version>4.1.67.Final</netty.version>
+        <netty.version>4.1.74.Final</netty.version>
```

Comment:
The official dependency of netty with vertx 4.1.8 is netty 4.1.72.Final.
I would suggest to stick to the version defined by vertx as we already encountered strange behavior when these 2 dependencies were not in sync.